### PR TITLE
WooCommerce Integration Implementation

### LINKED ISSUE #56: WooCommerce Integration Implementation

### PR DESCRIPTION
**Pull request by AtaAtasoy**

### CODE REVIEW

**Comment by AtaAtasoy** [comment]
The trivial functionalities for WooCommerce integration is implemented. 

We can create tags and categories by passing form-data to the corresponding endpoints. Refer to the WooCommerce doc for example tag and category data to pass to the functions.

**Comment by AtaAtasoy** [comment]
After closing this PR. I want to move the image annotation request from frontend to the image route in the backend. Then I could implement annotating the image on the WooCommerce server and updating the product with the annotation results.

File: api/flaskr/routes/woocommerce.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
+from flask import Blueprint, json, request, url_for, redirect, session, current_app, jsonify
+from flask.wrappers import Response
+from ..wcapi import woocommerce
+
+woocommerce_route = Blueprint('woocommerce_route', __name__)
+
+# Products
+@woocommerce_route.route("/createProduct", methods=['POST'])
```
Comment (Dogacel) [inline api/flaskr/routes/woocommerce.py:8]: Why .route ?

**Review by Dogacel — APPROVED** [review]

File: api/flaskr/routes/woocommerce.py (hunk 1 above)
Comment (AtaAtasoy) [reply api/flaskr/routes/woocommerce.py:8]: isn't it the way to add to add routes to a blueprint ?

Comment (Dogacel) [reply api/flaskr/routes/woocommerce.py:8]: I don't quite remember TBH.

File: api/flaskr/wcapi.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from woocommerce import API
+import os
+
+
+class woocommerce:
+    wcapi = API(
+        url="https://wooatasoy.wpcomstaging.com",
+        consumer_key = os.getenv("WOO_CONSUMER_KEY"),
+        consumer_secret = os.getenv("WOO_CONSUMER_SECRET"),
```
Comment (Dogacel) [inline api/flaskr/wcapi.py:9]: Would you like to move this into user information into mongodb? Maybe we can even  have a functional integrations page. ( So user can paste their keys / secrets there and will be saved into db )

**Review by Dogacel — CHANGES_REQUESTED** [review]

File: api/flaskr/wcapi.py (hunk 2 above)
Comment (AtaAtasoy) [reply api/flaskr/wcapi.py:9]: Ok, I'll create another issue and modify the integrations page to let the user log their consumer keys. It'll be an enhancement to the Integrations page.

**Review by Dogacel — APPROVED** [review]

**Pull request merged**